### ISSUE TITLE
Missing aria-controls value in list view pagination

### ISSUE THREAD

**Issue by seanredmond** [nypl-spacetime/maps-by-decade#94]
The pagination at the bottom of the list page has an empty `aria-controls` attribute. I imagine it should match the one at the top? `<ol aria-controls="paginate-current-page">`

**Comment by bertspaan** [comment]
Yes, it should! Fixing right now.
